.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "java.configuration.updateBuildConfiguration": "automatic",
+  "java.jdt.ls.vmargs": "-XX:+UseParallelGC -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xmx2G -Xms100m -javaagent:\"/home/randika/.vscode/extensions/vscjava.vscode-lombok-1.0.1/server/lombok.jar\"",
+  "workbench.colorTheme": "Solarized Light"
+}
```

Comment:
do we need this file? please check whether there are any other unwanted files

Reply:
removed